### ISSUE TITLE
Migration crashed with code

### ISSUE THREAD

**Issue by wortforfilms** [prisma/migrate#619]
Hi Prisma Team! Prisma Migrate just crashed. 

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | darwin             | 
| Node        | v10.16.0           | 
| Prisma CLI  | 2.10.0             | 
| Binary      | af1ae11a423edfb5d24092a85be11fa77c5e499c| 

## Error
```
Error: Error in migration engine.
Reason: [migration-engine/connectors/sql-migration-connector/src/sql_renderer/postgres_renderer.rs:213:26] AlterColumn on unknown column

Please create an issue in the migrate repo with
your `schema.prisma` and the prisma command you tried to use 🙏:
https://github.com/prisma/migrate/issues/new

```

**Comment by tomhoule** [comment]
Thanks for reporting this!

Can you share (parts of) your schema and/or migrations? This would help use reproduce the issue (I can't make it happen locally at the moment).

**Comment by wortforfilms** [comment]
Thanks for such quick response. I deleted old migrations and started again and issue was gone.

**Comment by tomhoule** [comment]
We worked on this problem and just released a fix in 2.10.1. Can you confirm that this does not happen anymore with the new version?

**Comment by tomhoule** [comment]
Other users confirmed 2.10.1 works for them, so let's close this issue. Please open a new one if necessary.
